react/features/base/react/components/native/Container.js

```diff
@@ -43,13 +44,22 @@ export default class Container extends AbstractContainer {
             return null;
         }
 
+        const viewProps = {
+            ...props
+        };
+
         const onClickOrTouchFeedback = onClick || touchFeedback;
+
+        if (!onClickOrTouchFeedback) {
+            viewProps.onLayout = onLayout;
+        }
+
         let element
             = super._render(
                 View,
                 {
                     pointerEvents: onClickOrTouchFeedback ? 'auto' : 'box-none',
-                    ...props
+                    ...viewProps
```

Comment:
Is `onLayout: onClickOrTouchFeedback ? undefined : onLayout` be sufficient?

Reply:
Absolutely, thanks!